packages/contracts-bridge/contracts/test/TokenRegistry.t.sol

```diff
@@ -464,7 +471,10 @@ contract TokenRegistryTest is BridgeTestFixture {
         bytes32 id = "It's over 9000";
         address repr = tokenRegistry.getRepresentationAddress(domain, id);
         assertEq(repr, address(0));
-        address calculated = computeCreateAddress(address(tokenRegistry), 2);
+        address calculated = computeCreateAddress(
```

Comment:
should this be a named utility function? `expectedTokenAddress` or something like that?

Reply:
TBH, I am not sold with adding all these utility functions in term of whether it makes the code more readable (albeit reducing code repeat).

Reply:
the reason I like utility functions is that it pushes the code towards even abstraction layers. When I'm reading a test, I don't want to break to remember how contract addresses are computed, or whether my registry uses `create` or `create2`. That is a different abstraction layer than the test functions on, and the details should be hidden when we're reading the test. Only things that work _on the same abstraction layer_ should be in the test

obv that's a bit of a muddy boundary, but clearly we're not interested in how addresses are calculated here